### DISCUSSION TITLE
Обход блокировки Hetzner

### DISCUSSION THREAD

**Discussion by Tymani24** [Flowseal/zapret-discord-youtube#848] [General]
Можно ли как-то обойти блокировку немецкого хостинг-провайдер Hetzner?
А то без VPN я не могу зайти на свой игровой сервер Rust, который находится в дата центре хостинг-провайдера Hetzner.

**Comment by amozebus** [comment]
@Tymani24 Если вы можете зайти только с VPN, скорее всего ресурс блокирует доступ с вашего IP. Для обхода такой блокировки нужен VPN. Zapret не имеет функционала VPN.

**Comment by belvikont** [comment]
Эй! Если ты ищешь способ защитить свои данные в интернете, у нас есть отличное решение — VPN! Это как невидимый щит для твоего устройства. С ним ты сможешь безопасно серфить в интернете, не беспокоясь о том, что кто-то может подглядеть за тобой. 

Плюс, он работает на всех твоих устройствах: будь то компьютер с Windows или Mac, или смартфон на Android или iOS. Так что, если хочешь наслаждаться свободой в интернете и быть уверенным в своей безопасности, загляни к нам! Это просто и удобно. Защити свои личные данные и серфинг станет еще приятнее! 
https://prohoster.info/vpn

**Comment by Dronatar** [comment]
Просто добавь `hetzner.com` например в `list-general`
